src/models/exercise.model.ts

```diff
@@ -0,0 +1,6 @@
+import { Set } from './set.model'
+
+export interface Exercise {
+  exerciseName: string;
+  sets: Set[];
+}
```

Comment:
We're already exporting an interface  called `exercise` (see src/lib/exercises/exercise.ts)`
Since this is specific to logging, we could rename it? Also, we can restrict the exerciseName to use only the exercises we defined by using the `ExerciseEnum`
```suggestion
export interface ExerciseLog {
  exerciseName: ExerciseEnum;
  sets: Set[];
}
```